runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonPlugin.java

```diff
@@ -24,9 +24,14 @@
  */
 package net.runelite.client.plugins.cannon;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
It appears this line is causing a CheckStyle violation. Did you intend to add this annotation somewhere?